frontend/src/components/common/form/FastCurrencyInput.test.tsx

```diff
@@ -1,17 +1,13 @@
 import { render } from '@testing-library/react';
-import Enzyme from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
 import { Form, Formik } from 'formik';
 import { noop } from 'lodash';
 import React from 'react';
 
 import { FastCurrencyInput } from './FastCurrencyInput';
 
-Enzyme.configure({ adapter: new Adapter() });
-
 describe('FastCurrencyInput', () => {
   it('fast currency input renders correctly', () => {
-    const { container } = render(
+    const { asFragment } = render(
```

Comment:
This is the recommended way to do snapshot testing with `react-testing-library`

Reply:
Ooh good to know